packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -2,12 +2,12 @@ import styled from 'styled-components';
 import { useCallback, useEffect, useRef, useState } from 'react';
 import { createCooldown } from '@trezor/utils';
 
-type Directions = 'top' | 'left' | 'right' | 'bottom';
+type Directions = 'top' | 'left' | 'right' | 'bottom' | null;
```

Comment:
I would rename it to singular:

`Directions -> Direction`

`Directions` should be in case of `Array<Direction>`

Reply:
Good point, thanks!